README.md

```diff
@@ -299,16 +299,16 @@ if(ParsePush::hasStatus($response)) {
         // handle a failed request
 
     }
-    
+
```

Comment:
its just trailing whitespace.  its easy to introduce trailing whitespace, especially with our fancy editors that auto indent....